<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>619</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>945</width>
    <height>619</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>945</width>
    <height>619</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Detector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>941</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>Видео</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>721</width>
           <height>471</height>
          </rect>
         </property>
         <property name="title">
          <string>Видеопоток</string>
         </property>
         <widget class="QLabel" name="videoSteam">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>691</width>
            <height>431</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>480</y>
           <width>721</width>
           <height>71</height>
          </rect>
         </property>
         <property name="title">
          <string>Информация</string>
         </property>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="geometry">
           <rect>
            <x>15</x>
            <y>20</y>
            <width>691</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>740</x>
           <y>190</y>
           <width>191</width>
           <height>191</height>
          </rect>
         </property>
         <property name="title">
          <string>Управление</string>
         </property>
         <widget class="QPushButton" name="startButton">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>20</y>
            <width>75</width>
            <height>44</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="stopButton">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>80</y>
            <width>75</width>
            <height>44</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="gui/pics/stop.png"/>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="makePhotoButton">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>140</y>
            <width>75</width>
            <height>44</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="gui/pics/camera.png"/>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>740</x>
           <y>10</y>
           <width>191</width>
           <height>161</height>
          </rect>
         </property>
         <property name="title">
          <string>Выбор камеры</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Выбор камеры</string>
          </property>
         </widget>
         <widget class="QComboBox" name="cameras">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="geometry">
          <rect>
           <x>740</x>
           <y>390</y>
           <width>191</width>
           <height>161</height>
          </rect>
         </property>
         <property name="title">
          <string>Клиент</string>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>20</y>
            <width>161</width>
            <height>131</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Архив</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>0</y>
           <width>521</width>
           <height>551</height>
          </rect>
         </property>
         <property name="title">
          <string>Результат поиска</string>
         </property>
         <widget class="QTableWidget" name="findResult">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>501</width>
            <height>521</height>
           </rect>
          </property>
          <property name="rowCount">
           <number>10</number>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>110</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>70</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>70</number>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>70</number>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Клиент</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Время прибытия</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Время убытия</string>
           </property>
          </column>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>391</width>
           <height>171</height>
          </rect>
         </property>
         <property name="title">
          <string>Поиск по дате</string>
         </property>
         <widget class="QDateTimeEdit" name="dateTimeFrom">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>50</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2021</year>
            <month>5</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>dd/mm/yy hh:mm:ss</string>
          </property>
         </widget>
         <widget class="QDateTimeEdit" name="dateTimeTo">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>50</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2022</year>
            <month>6</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="date">
           <date>
            <year>2022</year>
            <month>6</month>
            <day>1</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>dd/mm/yy hh:mm:ss</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>От</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>50</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>До</string>
          </property>
         </widget>
         <widget class="QPushButton" name="dateFindButton">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>120</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Поиск</string>
          </property>
          <property name="icon">
           <iconset theme="gui/pics/search.png"/>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>391</width>
           <height>351</height>
          </rect>
         </property>
         <property name="title">
          <string>Известные клиенты</string>
         </property>
         <widget class="QTableWidget" name="ourClientWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>371</width>
            <height>321</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Фото</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Старт</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Стоп</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
